src/utils/date_utils.cc

```diff
@@ -134,6 +146,8 @@ namespace Astroid {
       clock_format = ClockFormat::TWENTY_FOUR_HOURS;
     } else if (c_f == "12h") {
       clock_format = ClockFormat::TWELVE_HOURS;
+    } else if (c_f == "plain") {
+      clock_format = ClockFormat::PLAIN;
     } else if (c_f == "local") {
       clock_format = ClockFormat::LOCALE_DEFAULT;
     } else {
```

Comment:
error message should include 'plain' now, unless we go for changes in the other comment.